xml/System.Windows.Input/CommandManager.xml

```diff
@@ -567,7 +567,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Since this event is static, it will only hold onto the handler as a weak reference.  Objects that listen for this event should keep a strong reference to their event handler to avoid it being garbage collected. This can be accomplished by having a private field and assigning the handler as the value before or after attaching to this event.  
+ This event is implemented in such a way that it only holds onto the handler as a weak reference.  Objects that listen for this event should keep a strong reference to their event handler to avoid it being garbage collected. This can be accomplished by having a private field and assigning the handler as the value before or after attaching to this event.  
```

Comment:
Mentioning that it's implemented in such a way raises the question of how it's implemented, which is best avoided. It's much more straightforward here to simply say, "This event only hold on to the handler as a weak reference."

Reply:
@rpetrusha thanks for the feedback, I've updated the sentence. Indeed, my first reaction on the issue was to check the reference source :).